### PR TITLE
Remove all aggregator dependencies on k8s.io/kubernetes

### PR DESCRIPTION
**Pull request by deads2k**
This removes the last links to k8s.io/kubernetes for the aggregator.

@sttts last commit.

### CODE REVIEW

File: cmd/kube-aggregator/hack/update-codegen.sh [hunk 1]
```diff
@@ -67,3 +76,12 @@ ${informergen} \
   --listers-package k8s.io/kubernetes/cmd/kube-aggregator/pkg/client/listers \
   --output-package k8s.io/kubernetes/cmd/kube-aggregator/pkg/client/informers
   "$@"
+
+# this is a temporary hack until we manage to update codegen to accept a scheme instead of hardcoding it
+echo "rewriting imports"
+grep -Rl "\"k8s.io/kubernetes/pkg" "${KUBE_ROOT}/cmd/kube-aggregator/pkg/client" | \
```
Comment (sttts) [inline cmd/kube-aggregator/hack/update-codegen.sh]: no `-1` in BSD grep. This works: `grep -R -H <pattern> <dir> | cut -d: -f1 | sort | uniq`

File: cmd/kube-aggregator/pkg/apis/apiregistration/register.go [hunk 2]
```diff
@@ -20,7 +20,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	"k8s.io/apimachinery/pkg/runtime/schema"
-	kapi "k8s.io/kubernetes/pkg/api"
```
Comment (sttts) [inline cmd/kube-aggregator/pkg/apis/apiregistration/register.go]: add big fat comment please

**Comment by deads2k** [comment]
@k8s-bot unit test this

File: cmd/kube-aggregator/pkg/apis/apiregistration/register.go (hunk 2 above)
Comment (deads2k) [inline cmd/kube-aggregator/pkg/apis/apiregistration/register.go]: done

**Comment by deads2k** [comment]
comments addressed.

**Comment by sttts** [comment]
As discussed, the client-go scheme registration will go away when client-gen can handle that.

Based on this, lgtm.

**Comment by deads2k** [comment]
Doesn't affect external functionality or cross process concerns.  Isolated to leaf, retest-not-required.

**Pull request merged**